play-services-base/core/src/main/AndroidManifest.xml

```diff
@@ -5,10 +5,17 @@
   -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
+    <permission android:name="org.microg.gms.permission.READ_SETTINGS"
```

Comment:
`:play-services-base-core` is a public library used by third party apps. Permissions must be scoped to the application (via `${applicationId}`) to not cause conflicts.

Reply:
You also need to update the permission name used in the `<provider>`'s read/writePermission attribute. ;)

Reply:
Of course :disappointed: 4123bcf085f4ae0a8d468b701243e4b7f9c39826